data/map planets.txt

```diff
@@ -3404,6 +3703,21 @@ planet "Oup Je"
 	bribe 0
 	security 0
 
+planet O-Vasa-Oa
+	attributes wealthy successor urban
+	landscape land/hills4
+	description `This binary planet is the collective seat of what are known as the New Houses, the noble families who traveled through the wormhole to this region of space during the collapse of the Predecessors' empire. Many within the New Houses object to that moniker, arguing that their family lines are in fact the more extensive and noble and that they should be categorized by that, rather than their time of arrival.`
```

Comment:
I saw another planet with 5 required reputation that had the branching descriptions. Would they be necessary here as well?

Reply:
The many governments make this complicated, but: roughly 1/3 of the Successor planets are accessible before FL learns their language; these are the ones that have the branching descriptions. For this planet (and the other 2/3), the player does not gain the required reputation with the planet's government until _after_ they gain the translation device.